app/views/welcome/index.html.erb

```diff
@@ -1,11 +1,30 @@
+<br>
+
+
 <% content_for :top_content do %>
+  <CENTER>
```

Comment:
Try not to use `<center>` tags as these are a styling concern and a deprecated technique for centering.

To align text centrally in a heading:
```css
h2 {
  text-align: center;
}
```